addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -849,63 +745,117 @@ options.registry.facebookPage = options.Class.extend({
                 limit: 1,
             }).then(function (res) {
                 if (res) {
-                    self.fbData.href = res[0].social_facebook || 'https://www.facebook.com/Odoo';
+                    self.fbData.href = res[0].social_facebook || '';
                 }
             }));
         }
 
-        return Promise.all(defs);
-    },
-    /**
-     * @override
-     */
-    start: function () {
-        var self = this;
-        this.$target.on('click.facebook_page_option', '.o_add_facebook_page', function (ev) {
-            ev.preventDefault();
-            self.fbPageOptions();
-        });
-        return this._super.apply(this, arguments);
-    },
-    /**
-     * @override
-     */
-    destroy: function () {
-        this._super.apply(this, arguments);
-        this.$target.off('.facebook_page_option');
+        return Promise.all(defs).then(() => this._markFbElement());
     },
 
     //--------------------------------------------------------------------------
     // Options
     //--------------------------------------------------------------------------
 
     /**
-     * Opens a dialog to configure the facebook page options.
+     * Toggles a tab.
+     *
+     * @see this.selectClass for parameters
+     * @param {String} optionName the name of the option to toggle
+     */
+    toggleOption: function (previewMode, widgetValue, {optionName}) {
```

Comment:
I'd prefer you call the last argument `params` and use the object. Easier to grep if all options use the same argument names.